HelpSource/Classes/LFSaw.schelp

```diff
@@ -20,7 +20,8 @@ argument::iphase
 
 Initial phase offset. For efficiency reasons this is a value
 ranging from 0 to 2.
-
+Note:: enter "1" for an initial phase of 0 radiants.
+A value of 0 would start the cycle at pi radiants. See the example below.::
```

Comment:
I'm afraid I was taken by a logical error: the wave's first value is 0. Also like SinOsc etc. 
In my application I stumbled upon the mistake to treat LFSaw like a phasor, running from amplitude values 0 to 1.
But LFSaw is just a not bandlimited sawtooth oscillator.
So IMO the whole commit is pointless... 
(BTW, this commit is outdated: wrong example code!)